app/assets/stylesheets/components/maps/_v-map-header.scss

```diff
@@ -0,0 +1,20 @@
+.v-map-header {
+  background-color: $grey-black;
+  color: $grey-xlight;
+  display: flex;
+  font-family: $header-font;
+  font-size: rem-calc(25);
+  font-weight: 600;
+  justify-content: space-between;
+  line-height: $header-font-line-height;
+  padding: rem-calc(14 24);
+
+  .close-button {
+    margin-left: rem-calc(12);
+    &::after {
+      content: "⨯";
```

Comment:
In the styles there should be an actual icon for this - if you click the insect icon you should be able to download the assets.

Reply:
I tried using the included icon but the SVG itself (if not mistaken) has an embedded circle around the "X". Maybe I can retry using it, but I fear the icon will display too small relative to the header text because it is wrapped in a circle. That's why I opted for this instead.

Reply:
I found one that should work - I'll send ver slack as it doesn't want to upload here.